vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml

```diff
@@ -0,0 +1,105 @@
+---
+- name: Install required Ubuntu packages
+  block:
+    - name: Update all packages to their latest version
+      apt: update_cache=yes name="*" state=latest
+
+    - name: Fetch yarn gpg key
+      apt_key:
+        url: https://dl.yarnpkg.com/debian/pubkey.gpg
+        state: present
+
+    - name: Add yarn release key
+      lineinfile:
+        path: /etc/apt/sources.list.d/yarn.list
+        line: deb https://dl.yarnpkg.com/debian/ stable main
+        create: yes
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Add OS release key
+      lineinfile:
+        path: /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+        line: "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_{{ OS_VERSION_ID }}/ /"
+        create: yes
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Fetch OS release key
+      apt_key:
+        url: "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_{{ OS_VERSION_ID }}/Release.key"
+        state: present
+
+    - name: Update all packages to their latest version
+      apt: update_cache=yes name="*" state=latest
+
+    - name: Set update-alternatives to python3
+      community.general.alternatives:
+        name: python
+        path: /usr/bin/python3.6
```

Comment:
original script had specific versions for python (3.6.1 & 3.8.1), would not be a problem in this case?

Reply:
I think no, since it passed the CI. But I will add a condition. Thanks.

Reply:
Addition. 
It wasn't 3.8.1 but 3.8 1 previously. And this has failed when I added 3.8 1. That's why I made 3.6/3.8 instead of 3.6 1/3.8 1.

Reply:
I think it is fine to keep it like this @furkatgofurov7  ?

Reply:
My bad, I ignored reading the whole line and missed missing .(dot). I think it is fine as it is, sorry for the confusion.

Reply:
No worries. I had the same confusion too :).